docusaurus.config.js

```diff
@@ -135,10 +135,6 @@ const config = {
             sidebarId: "community",
             label: "Community",
           },
-          {
-            type: "localeDropdown",
-            position: "right",
-          },
```

Comment:
Is this desired?  It removes the language dropdown:
https://meag-navbar-css.avalanche-docs.pages.dev/

<img width="179" alt="Screenshot 2023-05-30 at 6 50 04 PM" src="https://github.com/ava-labs/avalanche-docs/assets/19595055/950c7b86-4242-4814-965c-76d2800fde16">

Reply:
Yes, we currently only support English so the dropdown is misleading. Support for Spanish and Chinese will likely be added in the future.